docs/fml-front-end-format.md

```diff
@@ -0,0 +1,198 @@
+---
+id: fml-front-end-format
+title: Feature Manifest Language Front-end Format
+slug: /fml-front-end-format
+---
+
+- Status: new
+- Deciders: @travis79, @jhugman, @teshaq, @k88hudson
+- Date: 2021-10-26
+
+## Context and problem statement
```

Comment:
Is the scope of this proposal just to use YAML or to propose a schema / format for the actual manifest?

Reply:
Just the selection of YAML, but as I just re-read the requirements of the issue in Jira, this should also specify the schema. I am working on the front end parsing implementation in parallel to this, and I hope to settle on the schema while working that out (in case there are some tricky things to represent). So I'll update this and add the schema definition ASAP. In hindsight, maybe I pinged you for review a little early, @k88hudson